### ISSUE TITLE
[BUG] - Attempted import error: 'calendar' is not exported from '@nextui-org/theme'

### ISSUE THREAD

**Issue by marketingcashfin** [nextui-org/nextui#2954]
### NextUI Version

2.3.6

### Describe the bug

Whenever I update the package `@nextui-org/react` or install `npm install @nextui-org/calendar`, I always get an error 

```
./node_modules/@nextui-org/calendar/dist/chunk-KHPJCHY4.mjs
Attempted import error: 'calendar' is not exported from '@nextui-org/theme' (imported as 'calendar').
```

Could you kindly help me fix this or if its a bug. I am upgrading from V.2.29
I have even installed `npm install @nextui-org/calendar`

### Steps to Reproduce the Bug or Issue

Upgrade from version 2.2.9 to latest version, of install `npm install @nextui-org/calendar`

### Expected behavior

Successfully import `import {DateRangePicker} from "@nextui-org/react";` and use it without getting error above

### Operating System Version

Windows

### Browser

Chrome

**Comment by monster99d2** [comment]
I am having the same issue.

**Comment by monster99d2** [comment]
FYI I was able to get this working. Once I reverted back to the older version (2.2.9), I ran in my project "npm upgrade" and it now seems to work.

**Comment by wingkwong** [comment]
Can you guys share how you upgraded it? also if possible please share your repo / stackblitz link for us to investigate.

**Comment by desmondinho** [comment]
Hello,

I've just fixed the same issue while importing the `DateRangePicker` component.

The error is saying that `calendar` is not exported from the `@nextui-org/theme`, I checked the exports, and it really wasn't, so I updated the package to the latest.

After that I got another error that was saying something is not exported from `@nextui-org/system`, so I updated this package to the latest too.

Everything is working now like a charm!
